content/docs/introduction/comparison.md

```diff
@@ -246,41 +246,36 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensuapp.org/) is broadly speaking a more modern Nagios.
+[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
 
 ### Scope
 
-The same general scope differences as in the case of
-[Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
+Sensu checks adhere to the [Nagios check specification](https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html). Sensu checks operate on two different models: publish and subscribe or standalone. It is possible to silence not just individual checks, but entire subscriptions. 
 
-The primary difference is that Sensu clients [register themselves](https://sensuapp.org/docs/0.27/reference/clients.html#what-is-a-sensu-client),
-and can determine the checks to run either from central or local configuration.
-Sensu does not have a limit on the amount of perfData.
+The primary difference is that Sensu clients [register themselves](https://docs.sensu.io/sensu-core/latest/reference/clients)
+and can determine the checks to run either from central or local configuration. Sensu clients can also be automatically deregistered using a number of [plugins](https://github.com/sensu-plugins/sensu-plugins-sensu) and [integrations](https://docs.sensu.io/sensu-enterprise/latest/built-in-handlers/)
```

Comment:
I'm not sure degegistration should be mentioned, that's kinda implied with this sort of system and we don't mention it for any of the others.